hadoop-ozone/recon/pom.xml

```diff
@@ -339,6 +339,12 @@
       <groupId>org.springframework</groupId>
       <artifactId>spring-jdbc</artifactId>
       <version>${spring.version}</version>
+      <exclusions>
```

Comment:
@okumin - can you also check if we can exclude this spring-jcl jar for recon-codegen ?

Reply:
@devmadhuu Yes. The dependency doesn't take effect now because of dependency management of maven but it is clear to exclude. I did so.
https://github.com/apache/ozone/pull/4544/commits/69c403a885039cc5c438deb2d6198366968afae6